src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
 // export function Person({ person }) {}
+import React from 'react';
+
+export const Person = ({ name, age, sex, isMarried, partnerName }) => (
+  <section className="person">
+    <h2 className="person__name">
+      {`My name is ${name}`}
+    </h2>
+    <p className="person__age">
+      {age && `I am ${age}`}
+    </p>
+    <p className="person__partner">
+      {(isMarried)
+        ? `My ${(sex === 'f') ? `husband's` : ` wife's`} name is ${partnerName}`
+        : `I am not married`
```

Comment:
better to prepare such strings before return - do not complicate your jsx by logic

Reply:
Ok thanks